### PR TITLE
[Dns] Rename dns recordset

### PR DESCRIPTION
**Pull request by dvbb**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/dns/Azure.ResourceManager.Dns/CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,64 @@
...
+using Microsoft.Rest;
+using Microsoft.Azure.Management.Dns;
+using Microsoft.Azure.Management.Dns.Models;
+```
+
+``` c#
+var tokenCredentials = new TokenCredentials("YOUR ACCESS TOKEN");
+DnsManagementClient dnsManagementClient = new DnsManagementClient(credentials);
+var iothub =await dnsManagementClient.Zones.CreateOrUpdateAsync
+                (
```
Comment (archerzz) [inline sdk/dns/Azure.ResourceManager.Dns/CHANGELOG.md:31]: indentation?

File: sdk/dns/Azure.ResourceManager.Dns/README.md [hunk 2]
```diff
@@ -6,54 +6,110 @@ This package follows the [new Azure SDK guidelines](https://azure.github.io/azur
 
 ### Install the package
 
-Install the Azure dns management library for .NET with [NuGet](https://www.nuget.org/):
+Install the Azure Dns management library for .NET with [NuGet](https://www.nuget.org/):
```
Comment (archerzz) [inline sdk/dns/Azure.ResourceManager.Dns/README.md:9]:
```suggestion
Install the Azure DNS management library for .NET with [NuGet](https://www.nuget.org/):
```
I think we don't need to follow csharp naming convention for English texts.

File: sdk/dns/Azure.ResourceManager.Dns/CHANGELOG.md [hunk 3]
```diff
@@ -1,12 +1,59 @@
 # Release History
 
-## 1.0.0-preview.2 (Unreleased)
+## 1.0.0-beta.1 (Unreleased)
 
+### Breaking Changes
 
-## 1.0.0-preview.1 (2020-09-23)
+New design of track 2 initial commit.
```
Comment (ArthurMa1978) [inline sdk/dns/Azure.ResourceManager.Dns/CHANGELOG.md:7]: Consider to provide some major changes of new track2:

### General New Features

- Support MSAL.NET, Azure.Identity is out of box for supporting MSAL.NET
- Support [OpenTelemetry](https://opentelemetry.io/) for distributed tracing
- HTTP pipeline with custom policies
- Better error-handling
- Support uniform telemetry across all languages

> NOTE: For more information about unified authentication, please refer to [Azure Identity documentation for .NET](https://docs.microsoft.com//dotnet/api/overview/azure/identity-readme?view=azure-dotnet)

File: sdk/dns/Azure.ResourceManager.Dns/src/autorest.md [hunk 4]
```diff
@@ -3,9 +3,46 @@
...
-require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/e0996d4f6dbca40ebf2fa4abf9a1cba45ada94d8/specification/dns/resource-manager/readme.md
- 
+library-name: Dns
+namespace: Azure.ResourceManager.Dns
+require: https://github.com/Azure/azure-rest-api-specs/blob/48a49f06399fbdf21f17406b5042f96a5d573bf0/specification/dns/resource-manager/readme.md
+clear-output-folder: true
+skip-csproj: true
+modelerfour:
+  lenient-model-deduplication: true
+
```
Comment (ArthurMa1978) [inline sdk/dns/Azure.ResourceManager.Dns/src/autorest.md:14]: Please copy the `rename-rules`

**Review by HarveyLink — APPROVED** [review]
LGTM

**Pull request merged**